content/docs/dvclive/api-reference/sklearn.md

```diff
@@ -0,0 +1,116 @@
+# Scikit-learn utilities
+
+See [detailed usage examples](/doc/dvclive/ml-frameworks/sklearn).
+
+## sklearn.log_confusion_matrix
+
+Generates a [_Confusion Matrix_](https://en.wikipedia.org/wiki/Confusion_matrix)
+plot (usable by `dvc plots`) .
+
+```py
+def log_confusion_matrix(y_true, y_pred, output_file):
+```
+
+#### Usage:
+
+```py
+from dvclive.sklearn import log_confusion_matrix
+
+y_true = [1, 1, 2, 2]
+y_pred = [2, 1, 1, 2]
+
+log_confusion_matrix(y_true, y_score, "cm.json")
+```
+
+### Description
+
+Stores the predictions and labels in the format expected by the DVC
+[confusion matrix template](/doc/command-reference/plots#example-confusion-matrix).
+
+### Parameters
+
+- `y_true` - Ground truth (correct) target values.
+
+- `y_score` - Estimated targets as returned by a classifier..
+
+- `output_file` - The name of the file where the `.json` plot will be saved.
```

Comment:
💅 The capitalization can work either way but in DVC docs we use lower case in these (up to you). The periods are not needed here though (incomplete sentences).
```suggestion
- `y_true` - ground truth (correct) target values

- `y_score` - estimated targets as returned by a classifier

- `output_file` - name of the file where the JSON plot will be saved
```